build/lib/install_go_versions.sh

```diff
@@ -53,6 +54,9 @@ GOBIN=${GOPATH}/go1.17/bin go install github.com/google/go-licenses@v1.2.1
 build::common::use_go_version "1.18"
 GOBIN=${GOPATH}/go1.18/bin go install github.com/google/go-licenses@v1.2.1
 
+build::common::use_go_version "1.19"
+GOBIN=${GOPATH}/go1.19/bin go install github.com/google/go-licenses@v1.2.1
```

Comment:
without this the make commands default to`go1.16`

Reply:
do we need to take care of it as mentioned here https://github.com/google/go-licenses/issues/128

Reply:
See the latest run, switching to 1.0.0 doesn't seem to have an effect on the CI run. I don't see this locally on my machine but I have the 1.0.0 version of the binary. Switching the local binary to 1.2.1 I am able to repro the issue.

Reply:
> Seems like it ends up using 1.16 when passing 1.19; this wasn't happening with 1.17. Any ideas @abhinavmpandey08 @jaxesn ?
> 
> ```
> /home/prow/go/src/github.com/aws/eks-anywhere-build-tooling/build/lib/gather_licenses.sh cluster-api-provider-nutanix /home/prow/go/src/github.com/aws/eks-anywhere-build-tooling/projects/nutanix-cloud-native/cluster-api-provider-nutanix/_output/ "." . "1.19"
> 321
> Adding /go/go1.16/bin to PATH
> 322
> vendor/sigs.k8s.io/json/json.go:24:2: //go:build comment without // +build comment
> 323
> make: *** [_output/attribution/go-license.csv] Error 1
> ```

This is fixed now by ensuring the right go version 1.19 has it's own case in the build scripts.